Session/Shared/BaseVC.swift

```diff
@@ -104,6 +105,10 @@ class BaseVC : UIViewController {
     deinit {
         NotificationCenter.default.removeObserver(self)
     }
+    
+    @objc func appDidBecomeActive(_ notification: Notification) {
+        // To be implemented
```

Comment:
This notification selector isn't doing anything - can we just revert the change?

Reply:
I noticed we have several view controllers adding the observer of `appDidBecomeActive` notification, just wanna to do that in `BaseVC` and clean up all the other ones. I'll make the code cleaner.

Reply:
Ah ok, no worries then